server/notebooks/templates/notebook.html

```diff
@@ -8,7 +9,8 @@
 </script>
 <div id="page"></div>
 <script>
-  window.userData = {{ user_info|safe }}
+  window.userData = {{ user_info|json }};
+  window.notebookInfo = {{ notebook_info|json }};
```

Comment:
@wlach: I'm made this change because I discovered that the old approach doesn't always produce valid JSON.  Particularly for booleans -- you get `True` (from Python) rather than `true` for JS/JSON.  `django-argonauts` seems built explicitly to solve this use case, but we could also do this ourselves if adding the dependency isn't preferred.

Reply:
On further inspection, I saw that some places were calling `json.dumps` to output these variables, and others weren't.  I still think this might be the better approach, but less critically -- if you'd rather I just make everything use `json.dumps` consistently instead, that's probably fine, though according to the argonaut docs, it is somewhat safer against script injection attacks -- if in fact such things aren't getting caught elsewhere in the stack.

Reply:
`django-argonauts` looks like an old and obsolete package to me. It seems that the latest version of django has a template builtin for our exact use case:

https://docs.djangoproject.com/en/2.1/ref/templates/builtins/#json-script

Maybe we can file an issue to convert our scripts to use that?